src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster.component.html

```diff
@@ -53,6 +53,13 @@
         </div>
         <div *ngSwitchCase="'3'"
              class="ml-5">
+          <h4 class="title"
+              i18n>Create Services</h4>
+          <br>
+          <cd-services [clusterCreation]="true"></cd-services>
```

Comment:
If we extend the Services interface according to what I suggested here, this should look like:

```suggestion
          <cd-services
            [hasDetails]="false"
            [hideServices]="['mon', 'mgr', 'crash']
            [hideColumns]="['running', 'status', 'last_refreshed']
          ></cd-services>
```